server/middlewares/validations/ReportValidation.js

```diff
@@ -0,0 +1,80 @@
+import Validator from 'validatorjs';
+
+/**
+ * @class ReportValidation
+ */
+class ReportValidation {
+  /**
+   * @description validate report input
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {func} next
+   *
+   * @return {void}
+   */
+  static validateReport(req, res, next) {
+    const { details, title } = req.body;
+
+    const data = {
+      title,
+      details,
+    };
+
+    const rules = {
+      title: 'required|max:80|min:3',
+      details: 'required|max:1000|min:3',
+    };
+
+    const message = {
+      'required.body': ':attribute field cannot be empty',
+      'max.body': 'The :attribute is too long. Max length is :max characters.',
+    };
```

Comment:
I would like to suggest you go through this validation again, I noticed you were writing validation messages for "body" which does not exist, I believe what you want to be writing a custom message for is for the title and details properties.